acceptance/tests/vault/vault_test.go

```diff
@@ -20,6 +21,11 @@ path "consul/data/secret/gossip" {
   capabilities = ["read"]
 }`
 
+	enterpriseLicensePolicy = `
+path "consul/data/secret/enterpriselicense" {
```

Comment:
Super minor nit, but I was thinking that this might read better as `enterprise-license` && `consul-enterprise-license` throughout. No changes required unless you feel the need :)

Reply:
^ I can see why you made this choice though since all of the previous examples are single words like `consul-server`, etc :)